SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

```diff
@@ -5,7 +5,7 @@ The HL7 Patient Identification (PID) segment requires a mapping from the <<acron
 .R8102
 [sdpi_requirement#r8102,sdpi_req_level=shall,sdpi_max_occurrence=2]
 ****
-The SDC patient context information shall only be mapped to the corresponding fields in the HL7 PID segment when the requirements (e. g. inferred patient ensemble context, associated patient context, etc.) as defined in the <<ref_ieee_11073_10700_2022>> are fulfilled.
+The SDC patient context information shall only be mapped to the corresponding fields in the HL7 PID segment when the requirements (e.g. inferred patient ensemble context, associated patient context, etc.) as defined in the <<ref_ieee_11073_10700_2022>> are fulfilled.

```

Comment:
It was important to Martin, that there are no e.g. statements in a requirement. Everything that is behind e.g. needs to move into a separate note.

Reply:
Ok, I can fix that here but be aware that this was only a correction of "e. g." to "e.g." - there might be other occurrences of examples in requirements...